pages/index.js

```diff
@@ -221,8 +220,8 @@ const home = ({ session }) => {
                   </div>
                 </div>
                 <div className="mt-6">
-                  <nav className="grid gap-y-8">
-                    {mobileMenu.map((item) => (
+                  <nav className="grid gap-y-6">
+                    {mobileMenu.map((item)=>(
```

Comment:
Let's keep the spaces.

```suggestion
                    {mobileMenu.map((item) => (
```

Reply:
+1